theano/sandbox/cuda/nvcc_compiler.py

```diff
@@ -141,6 +141,9 @@ def compile_args():
                 flags.append('-arch=sm_' + str(p['major']) +
                              str(p['minor']))
 
+        # Needed for glibc 2.23
+        flags.append('-D_FORCE_INLINES')
```

Comment:
Where is this flag defined? I don't like to use flags I don't know about. Why not fix this by adding include?

Also, I only find memcpy in the CPU code. Do you have the problem with your own op? If not, which op is causing this problem?